tests/unit/serialization/custom_global_serialization_test.py

```diff
@@ -128,3 +139,21 @@ def test_double_register_custom_serializer(self):
 
         with self.assertRaises(ValueError):
             service._registry.safe_register_serializer(TheOtherCustomSerializer, CustomClass)
+
+
+    def test_class(self):
+
+        config = _Config()
+
+        config.custom_serializers = {CustomClass: CustomSerializer, TheOtherCustomClass: CustomSerializer}
+
+        service = SerializationServiceV1(config)
+
+        data1 = service.to_data(CustomClass)
+        deserialized1 = service.to_object(data1)
+
+        data2 = service.to_data(TheOtherCustomClass)
+        deserialized2 = service.to_object(data2)
+
+        self.assertEqual(CustomClass, deserialized1)
+        self.assertEqual(TheOtherCustomClass, deserialized2)
```

Comment:
We need to add a test where we add a serializer for just `CustomClass`, and test serializing instances of `TheOtherCustomClass` (not the class, but actual objects of type `TheOtherCustomClass`). So, we can be sure that the change we did works fine

Reply:
It looks like there is a problem. 
```
Expected :<tests.unit.serialization.custom_global_serialization_test.TheOtherCustomClass object at 0x1110d4b20>
Actual   :<tests.unit.serialization.custom_global_serialization_test.CustomClass object at 0x1110d4af0>
```

Reply:
That looks fine to me, because we have used the serializer of a superclass, so it is expected to receive instances of a superclass in this case. Not the most intuitive design, but at least it is the same as Java